pkgs/development/python-modules/scitools/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib, fetchurl, python, buildPythonPackage, numpy, matplotlib}:
+
+buildPythonPackage rec {
+  pname = "scitools";
+  version = "0.9.0";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/hplgit/scitools/archive/scitools-${version}.tar.gz";
+    sha256 = "1aj3i56aim542krynza46plxbci7dfhq9rq7hwfw4946rjbm3klc";
+  };
+
+  buildInputs = [ matplotlib ];
+  propagatedBuildInputs = [ numpy ];
+
+  meta = {
```

Comment:
Add the license under meta here with `license = stdenv.lib.licenses.bsd3;`. And also add yourself as maintainer, `maintainers = with stdenv.lib.maintainers; [ jamtrott ];` after adding yourself to https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix.